File: episode/01.introduccion/19-ficheros/entrega9ficheros.py
# ASGMT files
# asigna valores iniciales
filename = 'points.txt'

# code
f = open(filename)
distances = []
for line in f:
    items = line.split()
    x = int(items[0])
    y = int(items[1])
    distance = (x**2 + y**2) ** 0.5
    distances.append(distance)

f.close()

# min_distance = 4.123105625617661
min_distance = min(distances)

print(min_distance)

File: episode/01.introduccion/19-ficheros/entrega9ficheros.rb
#!/usr/bin/env ruby
# asigna valores iniciales
filename = 'points.txt'

# code
distances = []

File.foreach(filename) do |line|
  items = line.split
  x = items[0].to_i
  y = items[1].to_i
  distance = (x**2 + y**2)**0.5
  distances.append distance
end

# min_distance = 4.123105625617661
min_distance = distances.min
puts min_distance
